pom.xml

```diff
@@ -19,8 +19,8 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-		<omero.version>5.4.10-ice36-b105</omero.version>
-		<bioformats.version>5.8.2</bioformats.version>
+<!--		<omero.version>5.4.10-ice36-b105</omero.version> -->
+<!--		<bioformats.version>5.8.2</bioformats.version> -->
```

Comment:
There are a lot of commented out lines in this pom now. If they are not used anymore, it is customary to remove them.

Reply:
yes, I will clean it up